tests/console/sudo.pm

```diff
@@ -63,7 +63,7 @@ sub full_test {
     assert_script_run 'id -un|grep ^bernhard';
     sudo_with_pw 'sudo id -un', grep => '^root';
     # I/O redirection; the redirection happens as user, not in sudo context, so should fail
-    sudo_with_pw 'sudo echo 2 >/run/openqa_sudo_test';
+    sudo_with_pw 'sudo echo 2 >/run/openqa_sudo_test', {expected_screen => 'user-console'};
```

Comment:
IMO this shouldn't even use `sudo_with_pw`. Just `script_run` and assert that it failed. It won't even attempt to run `sudo`.

Reply:
Good point. To actually ensure that this fails I would use `assert_script_run '! sudo echo 2 >/run/openqa_sudo_test';`. Deal?

Reply:
Done

Reply:
If it works, sure